tests/ImageSharp.Tests/project.json

```diff
@@ -20,9 +20,24 @@
     }
```

Comment:
Happy that you did not split up the test project. Maybe others feel different but I think we should keep all the tests in one project.